### ISSUE TITLE
Define features list for first prototype

### ISSUE THREAD

**Issue by robguinness** [pondenome/prototype#2]
First features to be prototyped:

- User registration and 2FA-based verification
- Adding friends
- Viewing feed
- Posting a message

**Comment by mwernerds** [comment]
If pondemone is going to be a social network, we need more ways of adding
contacts:
- user A connects his friends B and C, possibly by posting a PM to both of
them.

User story: Rob goes out for beer with Martin and Alice. He takes a selfie
of us three and sends it to us two. Alice and Martin can decide to accept
ignore or decline this offer.

Gatherings: users are physically somewhere (GPS Wi-Fi time) and connect
themselves with a gathering basically writing themselves into an address
book of the gathering. Knowing the key and while being there, users can
read this list. The proof of physical presence is difficult, but could be
done via blockchain, p2p, or a barcode presented.

User story: we all go to a conference and want to browse the attendees that
opted in by scanning the code. Finding friends comes at the cost of being
open for friends.

Please tell me, what you think about these additional mechanisms.

Best regards from Germany!
   Martin

Rob Guinness <notifications@github.com> schrieb am Mi., 23. Mai 2018, 15:30:

**Comment by craigtaub** [comment]
@robguinness should we add "profile page" (+ "sign in") to feature list?
I can add to project if u agree, wasnt sure if there was a reason had left these out. Perhaps to limit scope of work for prototype, but my thinking is if we get those first features done at a good pace we would probably want to keep expanding the functionality.

**Comment by robguinness** [comment]
Yes, these absolutely have to be a part of the first prototype! Feel free to add.
